### PR TITLE
docs(blockchain): Add blockchain package documentation

### PR DESCRIPTION
**Pull request by skambalin**
- Add source code documentation
- Generate MD docs

### CODE REVIEW

**Review by idmitriev — APPROVED** [review]

File: packages/blockchain/README.md [hunk 1]
```diff
@@ -1,8 +1,6 @@
-# @cere-ddc-sdk/blockchain
+# @cere-ddc-sdk/file-storage
```
Comment (upalinski) [inline packages/blockchain/README.md:1]: why it was changed to file-storage?

Comment (skambalin) [reply packages/blockchain/README.md:1]: Copy & past artifact. Will fix.

Comment (skambalin) [reply packages/blockchain/README.md]: Fixed. Thanks for noticing

File: packages/blockchain/src/DDCClustersPallet.ts [hunk 2]
```diff
@@ -9,28 +9,115 @@ import type {
...
+   * @example
+   *
+   * ```typescript
+   * const clusterId = '0x...';
+   * const nodeKeys = await blockchain.ddcClustersPallet.filterNodeKeysByClusterId(clusterId);
+   *
+   * console.log(nodeKeys);
+   * ```
+   */
   async filterNodeKeysByClusterId(clusterId: ClusterId) {
```
Comment (upalinski) [inline packages/blockchain/src/DDCClustersPallet.ts:61]: getClusterNodes? or getNodes or getNodesByClusterId? 

minor, just name looks a little bit unusual for library API

Comment (skambalin) [reply packages/blockchain/src/DDCClustersPallet.ts:61]: ¯\_(ツ)_/¯

**Pull request merged**